src/components/view/ConnectEhrView/ConnectEhrView.tsx

```diff
@@ -40,21 +41,28 @@ export default function ConnectEhrView(props: ConnectEhrViewProps) {
     return (
         <Layout colorScheme={props.colorScheme ?? "auto"}>
             {props.presentation &&
-            <NavigationBar title={title}
-                           showBackButton={props.presentation == "Push"}
-                           showCloseButton={props.presentation == "Modal"}/>
+                <NavigationBar title={title}
+                    showBackButton={props.presentation == "Push"}
+                    showCloseButton={props.presentation == "Modal"} />
             }
             {!props.presentation &&
-            <StatusBarBackground />
+                <StatusBarBackground />
             }
             <TextBlock>
-                {language("ehr-intro")}
+                {props.introText ?? language('ehr-intro-research')}
```

Comment:
The default never shows. introText, when left off the props, is carrying in as ''